### PR TITLE
Feature/rr 223 setup api endpoint to receive latest income details

### PR DESCRIPTION
**Pull request by CaraHill**
This is a pretty big change to the api - it allows both the api Rebate Forms Controller and admin Rebate Forms Controller to receive all of the fields that we need.

There is a new service to handle both creating and updating rebate_forms, and tests have been updated to work with this. I've also done manual tests using Postman and a local dev environment to ensure that it is working.

Some things to note:
- the calculator isn't working so testing could only be done to a point
- only one working test has been added for the new service - these should be added to to handle edge cases
- a helper method was marked commented out and it's test marked as the code it refers to will be changing once @mischa-s 's work lands.

### CODE REVIEW

File: app/services/rebate_forms_service.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class RebateFormsService
+  def initialize(rebate_form_attributes)
+    @rebate_form_attributes = rebate_form_attributes
+  end
+
+  def update
```
Comment (mischa-s) [inline app/services/rebate_forms_service.rb]: :sparkles:

File: app/services/rebate_forms_service.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
```
Comment (mischa-s) [inline app/services/rebate_forms_service.rb]: Love how clear and legible this file is :fireworks:
